pipelines/managed/push-disk-images-to-cdn/README.md

```diff
@@ -20,6 +20,12 @@ Tekton Pipeline to push disk images to a cdn using pulp
 | taskGitUrl                      | The url to the git repo where the release-service-catalog tasks to be used are stored                  | Yes | https://github.com/konflux-ci/release-service-catalog.git |
 | taskGitRevision                 | The revision in the taskGitUrl repo to be used                                                         | No       | -               |
 
+## Changes in 1.6.0
+* Update all tasks that now support trusted artifacts to specify the stepAction* parameters
```

Comment:
Is the single asterisk in _stepAction_ intentional? Just wondering it you wanted somehow to highlight it or if it was a typo. Same for all the other occurences.

Reply:
Maybe changing it to `stepAction(A|B)` - regex style. Maybe I am  the only one that got confused by it, although ater you said, it makes sense.